test/e2e/swaps/shared.js

```diff
@@ -63,8 +62,137 @@ const buildQuote = async (driver, options) => {
   await driver.clickElement('.searchable-item-list__primary-label');
 };
 
+const reviewQuote = async (driver, amount, swapFrom, swapTo) => {
+  await driver.clickElement({ text: 'Review swap', tag: 'button' });
+  await driver.waitForSelector({ text: 'Swap', tag: 'button' });
+
+  await driver.waitForSelector({
+    css: '[class*="box--align-items-center"]',
+    text: 'Estimated gas fee',
+  });
+
+  const sourceValue = await driver.waitForSelector(
+    '.main-quote-summary__source-row-value',
+  );
+  assert.equal(
+    await sourceValue.getText(),
+    amount,
+    'Error: Quote has wrong amount',
+  );
+
+  const sourceSymbol = await driver.waitForSelector(
+    '.main-quote-summary__source-row-symbol',
+  );
+  assert.equal(
+    await sourceSymbol.getText(),
+    swapFrom,
+    'Error: SwapFrom has wrong symbol',
+  );
+
+  const swapToSymbol = await driver.waitForSelector(
+    '.main-quote-summary__destination-row',
+  );
+  assert.equal(
+    await swapToSymbol.getText(),
+    swapTo,
+    'Error: SwapTo has wrong symbol',
+  );
+
+  await driver.waitForSelector(
+    '[class="exchange-rate-display main-quote-summary__exchange-rate-display"]',
+  );
+
+  await driver.waitForSelector('[class="fee-card__info-tooltip-container"]');
+
+  await driver.waitForSelector({
+    css: '[class="countdown-timer__time"]',
+    text: '0:23',
+  });
+};
+
+const waitForTransactionToComplete = async (driver, tokenName) => {
+  const sucessfulTransactionMessage = await driver.waitForSelector(
+    {
+      css: '[class="awaiting-swap__header"]',
+      text: 'Transaction complete',
+    },
+    { timeout: 30000 },
+  );
+
+  assert.equal(
+    await sucessfulTransactionMessage.getText(),
+    'Transaction complete',
+    'Incorrect transaction message',
+  );
+  const sucessfulTransactionToken = await driver.waitForSelector({
+    css: '[class="awaiting-swap__amount-and-symbol"]',
+    text: tokenName,
+  });
+  assert.equal(
+    await sucessfulTransactionToken.getText(),
+    tokenName,
+    'Incorrect token name',
+  );
+
+  await driver.clickElement({ text: 'Close', tag: 'button' });
+  await driver.waitForSelector('[data-testid="home__asset-tab"]');
+};
+
+const checkActivityTransaction = async (
```

Comment:
Same comment as the other one:
```
const checkActivityTransaction = async(driver, {
  index,
  amount,
  swapFrom,
  swapTo,
});

```

Reply:
Good suggestion. I have changed the code